.github/workflows/main.yml

```diff
@@ -146,10 +146,15 @@ jobs:
       - name: Install Ansible 2.9 compatible molecule and Jinja
         run: pip install --upgrade 'molecule[docker]<4.0' 'Jinja2<3.1'
         if: matrix.ansible == 'v2.9.17'
+      - name: Install collection dependencies
+        run: cd roles/backup && molecule dependency -s debian
```

Comment:
Looking at https://ansible.readthedocs.io/projects/molecule/usage/ I wonder if `-s` should be written as `--scenario-name`.

Reply:
How dare you taking my "I DEMAND LONGOPTS EVERYWHERE" and point it at me?! :)

Reply:
It's Monday, it shouldn't be easy today.